static/app/components/events/eventEntry.tsx

```diff
@@ -158,6 +158,11 @@ function EventEntry({
         />
       );
     case EntryType.SPANS:
+      // XXX: We currently do not show spans in the share view,
```

Comment:
I forgot what 'XXX' means 😄

Reply:
Just a way to call attention to a comment